tools/log_rotate/install_script.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Create directory, which keeps all logrotate-config to rotate hourly.
+mkdir /etc/logrotate.hourly.d
```

Comment:
Though it's rare but it's possible that this directory already exist, so can we use mkdir -p option here?

Reply:
Done.